site/en/blog/colrv1-fonts/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: COLRv1 Color Gradient Vector Fonts in Chrome 98
+description: >
+  In Chrome 98, the Chrome and Fonts teams have added support for COLRv1, an
+  evolution of the COLRv0 font format intended to make color fonts widespread by
+  adding gradients, compositing and blending, and improved internal shape reuse
+  for crisp and compact font files that compress well.
+layout:
+  'layouts/blog-post.njk'
+date: 2022-01-06
+authors:
+  - drott
+  - rsheeter
+hero: 'image/roKb5a4ddjOSoIbxRFI9kDeuVvE2/NQGoL7CmejHN0lUntQfz.png'
+alt: >
+  A sharp vs. a slightly blurry national park emoji and a bar chart highlighting
+  the size difference between COLRv1 and a bitmap font: about 9MB vs. 1.85MB.
+tags:
+  - chrome-98
+  - fonts
+---
+
+*Compact, compression-friendly, color vector fonts with all your favorite gradient flavors*
+
+In Chrome 98, the Chrome and Fonts teams have added support for COLRv1, an
+evolution of the COLRv0 font format intended to make color fonts widespread by
+adding gradients, compositing and blending, and improved internal shape reuse
+for crisp and compact font files that compress well.
+
+{% Aside 'caution' %}
+Examples in this blog post require a browser that supports
+COLRv1 fonts. Try viewing this post in Chrome 98 or above.
+{% endAside %}
+
+## Color Now
+
+On the web, text is generally drawn in a color specified in CSS. The font does
+not define any particular color, it just indicates where pixels should be
+placed. That's usually a good thing, CSS lets the author flexibly choose a
+color. However, sometimes a glyph has multiple colors and those colors have
+meaning. For example, this flag {% Img
+src="image/roKb5a4ddjOSoIbxRFI9kDeuVvE2/8hQhmhh6F8Ff7kquy4mB.png",
+alt="Transgender flag consisting of pale blue and pale pink stripes.",
+height="16", width="16" %} with light blue, pink, and white stripes would not convey the
+same meaning if it was simply drawn in the current text color.
+
+Today, for most users, emoji are the only color fonts they see. Emoji typically
```

Comment:
I am afraid I don't fully understand your explanations in this case. Wha are the rules for loan words? Sure it's a loan word from Japanese, not from Latin - that's clear. Are you saying loan words in English adopt the English plural form with a trailing s? Are you referring to the Latin plural _-i_ ending for Latin's second declension not applying here? 

The word "Emoji" being a loan word from Japanese would it not be correct to use the Japanese plural form "emoji" without ending on _"s"_ then? 

Emojipedia tried to do [some stats](https://blog.emojipedia.org/emojis-on-the-rise-as-plural/#:~:text=%22Emoji%22%20remains%20the%20preferred%20plural,rules%20to%20make%20them%20fit.) and sees "emoji" as the plural form that's most widely used. 

Anyways, no strong opinion either way - changed to emojis.

Reply:
Loan words don't generally keep their source language grammar. If they did, I don't think it would scale well. Some Latin terms are given an exception only because some of our ancestors believed Latin was a 'superior' language. 

As far as adapting load words to English, you already do this whether you know it or not. 'Wiki' is an adjective in Hawaiian. Do you say 'wiki' or do you say 'wiki site'? Quick! What's the plural of bazaar? I'll bet 'bazarha' never entered your mind even though that's what it was in Persian.

Reply:
(People now generally follow the Latin thing because that's what they've been taught. That it was 'superior' is how the practice got started originally.)

Reply:
Ack, thanks for the additional explanations.